pkg/util/deployment/deployment.go

```diff
@@ -14,15 +14,34 @@ const (
 	Production Mode = iota
 	Integration
 	Development
+
+	strDevelopment = "development"
+	strIntegration = "int"
+	strProduction  = "production"
 )
 
 func NewMode() Mode {
 	switch strings.ToLower(os.Getenv("RP_MODE")) {
-	case "development":
+	case strDevelopment:
 		return Development
-	case "int":
+	case strIntegration:
 		return Integration
+	case strProduction:
+		fallthrough
```

Comment:
hmm, we use "" == production (we default to production mode); from my perspective this change just confuses / is unnecessary extra text